docs/pages/components/SearchField.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: SearchField
+description: A form element that allows users to type in text for searching.
+source: https://github.com/dequelabs/cauldron/tree/develop/packages/react/src/components/SearchField/index.tsx
+---
+
+import { SearchField, FieldWrap } from '@deque/cauldron-react';
+
+```js
+import { SearchField } from '@deque/cauldron-react';
+```
+
+<Note>
+  Use [FieldWrap](./FieldWrap) to wrap all input components. The `SearchField`
+  component has been designed specifically to only be used when rendered as a
+  child of this component, and may not be completely accessible if rendered
+  outside of `FieldWrap`.
+</Note>
+
+## Examples
+
+### Resting
+
+```jsx example
+<FieldWrap>
+  <SearchField label="Resting" />
+</FieldWrap>
+```
+
+### Disabled
+
+```jsx example
+<FieldWrap>
+  <SearchField label="Disabled" disabled />
+</FieldWrap>
+```
+
+### Label Hidden
+
+```jsx example
+<FieldWrap>
+  <SearchField label="Hidden" hideLabel />
+</FieldWrap>
+```
+
+### Usage in Form
+
+By default, SearchField is wrapped in a `<form role="search">` to provide better accessibility hints to users of assistive technology. If your search field needs to be placed inside of an existing form, you must use isForm="false".
+
+```jsx example
+<form role="search">
+  <FieldWrap>
+    <SearchField label="Nested in Form" isForm={false} />
+  </FieldWrap>
+</form>
+```
+
+## Props
+
+<ComponentProps
+  className={true}
+  refType="HTMLInputElement"
+  props={[
+    {
+      name: 'label',
+      type: 'React.ReactNode',
```

Comment:
This should be using `ContentNode`, see: 
https://github.com/dequelabs/cauldron/blob/b7bf54c6c627676834f383c78bd31aaf9d6ace0e/packages/react/src/components/Combobox/Combobox.tsx#L36